### PR TITLE
Pinned messages drawer comments

### LINKED ISSUE #16875: Design review: pinned message drawer
- [x] 1.  Title is too big - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=3709:473614&mode=design#511498551)  
- [x] 2. Margins - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=3709:473614&mode=design#511498899)  
- [x] 3. Spacing between name and message doesn't match designs - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=3709:473614&mode=design#511500300)
- [ ] 4.Too big icons - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=3709:473614&mode=design#511501291)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #16875

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/menus/pinned_messages/style.cljs [hunk 1]
```diff
@@ -1,8 +1,9 @@
 (ns status-im2.contexts.chat.menus.pinned-messages.style
   (:require [quo2.foundations.colors :as colors]))
 
-(def heading
-  {:margin-horizontal 20})
+(defn heading [community]
```
Comment (flexsurfer) [inline src/status_im2/contexts/chat/menus/pinned_messages/style.cljs]: community?

**Review by flexsurfer — COMMENTED** [review]
whats the point of changing icons? looks the same

File: src/status_im2/contexts/chat/menus/pinned_messages/style.cljs (hunk 1 above)
Comment (ibrkhalil) [reply src/status_im2/contexts/chat/menus/pinned_messages/style.cljs]: Fixed

**Comment by ibrkhalil** [comment]
One of the design comments was that the icon had a different size than Figma, I thought there was a difference in the file and when git told me they are different I thought sizing is different.

But turned out to be only a color difference 😄

**Review by flexsurfer — APPROVED** [review]

**Review by Francesca-G — APPROVED** [review]
Looks good to me :)

Adding follow up due to the missing community picture in the tag:
Screenshot 2023-08-23 alle 10 23 17

design reference:
Screenshot 2023-08-23 alle 10 25 22

**Comment by VolodLytvynenko** [comment]
Hi @ibrkhalil PR can be merged. e2e failures are not related to current PR

**Comment by ibrkhalil** [comment]
Added the community icon on the drawer
image

**Review by Francesca-G — APPROVED** [review]

**Comment by churik** [comment]
@ibrkhalil 
 did you create an issue follow-up based on the design review?

Based on https://github.com/status-im/status-mobile/blob/develop/doc/pipeline_process.md > Design review, all follow-up issues have to be created when PR is merged and linked to the original PR.

**Comment by ibrkhalil** [comment]
> @ibrkhalil did you create an issue follow-up based on the design review?
> 
> Based on https://github.com/status-im/status-mobile/blob/develop/doc/pipeline_process.md > Design review, all follow-up issues have to be created when PR is merged and linked to the original PR.

I addressed the review comments by Francesca in [this](https://github.com/status-im/status-mobile/pull/16998#issuecomment-1694261463) comment and when she approved I took it as it was correctly fixed/addressed.

**Pull request merged**